scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -315,6 +321,20 @@ public void initActivity(Body body) {
             logger.debug("Booting jmx...");
             this.jmxHelper.boot(authentication);
 
+            privateStore = PAActiveObject.newActive(AOSynchronization.class,
+                                                    new Object[] { PASchedulerProperties.getAbsolutePath(PASchedulerProperties.SCHEDULER_SYNCHRONIZATION_DATABASE.getValueAsString()) });
+
+            privateStore.isStarted();
```

Comment:
I suppose you do that to wait for the initActivity to be finished, correct? We could take this opportunity to log a message when this returns.

Reply:
ok